### ISSUE TITLE
[Bug] texture error when using sharp transparency mode

### ISSUE THREAD

**Issue by adfriz** [leezer3/OpenBVE#748]
### Description

when choosing sharp option the texture will broke, this seems only broke for shinkansen only.

### Reproduction
load extension cfg or its csv object and use sharp transparent option

## Route

no route

## Train

shinkansen E5 by ginga81

## Logs

no logs

### Related information 

* Operating system windows 10 20H2 64 bit
* Method of control Keyboard and mouse
* openbve 1.8.2.3 zip ver
![a](https://user-images.githubusercontent.com/76892624/156303195-48315bf6-fd56-489e-ba63-3ae088ce0739.jpg)
![b](https://user-images.githubusercontent.com/76892624/156303202-a662ff0c-2b77-459b-b222-334f414fbf6e.jpg)

**Comment by leezer3** [comment]
This is semi-expected. It's not quite a bug, but rather a 'feature' of the sharp transparency quality.

The train uses an alpha channel in it's textures, and the chassis has two stacked alpha channels. Blending between these in a single-pass doesn't work well (for somewhat complicated mathematical reasons), which is what sharp does.

Sharp transparency isn't reccomened unless you're really struggling for FPS.

**Comment by adfriz** [comment]
Oh ok i understand now why sharp mode behave like that.

Anyway should the option changed to Low, medium and high ? Instead sharp, intermediate and smooth.

**Comment by leezer3** [comment]
Descriptively in terms of what it does, sharp is absolutely correct- It's an absolute value from either the front or rear pixel, wheras smooth is blended between the two.

https://learnopengl.com/Advanced-OpenGL/Blending

Low isn't quite right, as if we were for example attempting to replicate exactly a BVE2 experience, sharp is specifically what we want there, as that never supported alpha.

TLDR:
It's complex :P

**Comment by adfriz** [comment]
Yeah, just reading that how blending works make me feel a bit dizzy 😅

Anyway thanks leezer3
